subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -532,16 +582,35 @@ static void isr_rx(void *param)
 	if (!radio_tmr_aa_restore()) {
 		const struct lll_sync_iso_stream *sync_stream;
 		uint32_t se_offset_us;
-		uint8_t se;
 
 		crc_ok_anchor = crc_ok;
 
 		sync_stream = ull_sync_iso_lll_stream_get(lll->stream_handle[0]);
-		se = ((lll->bis_curr - sync_stream->bis_index) *
-		      ((lll->bn * lll->irc) + lll->ptc)) +
-		     ((lll->irc_curr - 1U) * lll->bn) + (lll->bn_curr - 1U) +
-		     lll->ptc_curr + lll->ctrl;
-		se_offset_us = lll->sub_interval * se;
+
+		if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_ISO_SEQUENTIAL) &&
+		    (lll->bis_spacing >= (lll->sub_interval * lll->nse))) {
+			uint8_t se;
+
+			se = ((lll->bis_curr - sync_stream->bis_index) *
+			      ((lll->bn * lll->irc) + lll->ptc)) +
+			     ((lll->irc_curr - 1U) * lll->bn) +
+			     (lll->bn_curr - 1U) + lll->ptc_curr + lll->ctrl;
+			se_offset_us = lll->sub_interval * se;
+		} else if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_ISO_INTERLEAVED) &&
+			   (lll->bis_spacing < (lll->sub_interval * lll->nse))) {
+			uint8_t se;
+
+			se = (lll->bis_curr - sync_stream->bis_index) +
+			     ((((lll->irc_curr - 1U) * lll->bn) +
+			       (lll->bn_curr - 1U) + lll->ptc_curr) *
+			      lll->num_bis) + lll->ctrl;
+			se_offset_us = lll->bis_spacing * se;
```

Comment:
Suggest to use additional local variable to make this more readable. This is nearly impossible for a person to parse :D